ui/tests/acceptance/client-history-test.js

```diff
@@ -272,7 +273,33 @@ module('Acceptance | clients history tab', function (hooks) {
     assert.dom(SELECTORS.activeTab).hasText('History', 'history tab is active');
     // Message changes depending on ent or OSS
     assert.dom(SELECTORS.emptyStateTitle).exists('Empty state exists');
-    assert.dom('[data-test-popup-menu-trigger="month"]').exists('Dropdown exists to select month');
-    assert.dom('[data-test-popup-menu-trigger="year"]').exists('Dropdown exists to select year');
+    assert.dom(SELECTORS.monthDropdown).exists('Dropdown exists to select month');
+    assert.dom(SELECTORS.yearDropdown).exists('Dropdown exists to select year');
+  });
+
+  test('shows error template if permissions denied querying activity response with no data', async function (assert) {
+    this.server = new Pretender(function () {
+      this.get('/v1/sys/license/status', () => sendResponse(null, 403));
+      this.get('/v1/sys/version-history', () => sendResponse(null, 403));
+      this.get('/v1/sys/internal/counters/config', () => sendResponse(null, 403));
+      this.get('/v1/sys/internal/counters/activity', () => sendResponse(null, 403));
+      this.get('/v1/sys/health', this.passthrough);
+      this.get('/v1/sys/seal-status', this.passthrough);
+      this.post('/v1/sys/capabilities-self', this.passthrough);
+      this.get('/v1/sys/internal/ui/feature-flags', this.passthrough);
+    });
```

Comment:
No need to change but a suggestion for the future is Mirage might be better in this situation since there are some common endpoints between the `client-history` and `client-current` tests. By creating a mirage handler you could share it between tests and all the endpoints would be defined in one place along with the `sendResponse` function.